gnocchi/rest/aggregates/processor.py

```diff
@@ -46,33 +39,39 @@ def __init__(self, reason):
 class MetricUnaggregatable(Exception):
     """Error raised when metrics can't be aggregated."""
 
-    def __init__(self, metrics, reason):
-        self.metrics = metrics
+    def __init__(self, metrics_and_aggregations, reason):
+        self.metrics_and_aggregations = metrics_and_aggregations
         self.reason = reason
+        metrics = ("%s/%s" % (m.id, a) for (m, a) in metrics_and_aggregations)
         super(MetricUnaggregatable, self).__init__(
-            "Metrics %s can't be aggregated: %s"
-            % (", ".join((str(m.id) for m in metrics)), reason))
+            "Metrics %s can't be aggregated: %s" % (
+                ", ".join(metrics), reason))
+
+
+def _get_measures_timeserie(storage, metric, aggregation, ref_identifier,
```

Comment:
i guess that'll be needed later? :)

Reply:
I have put explanation in the commit msg !